src/pykeen/experiments/nodepiece/galkin2022_nodepiece_codexl.yaml

```diff
@@ -0,0 +1,43 @@
+metadata:
+  title: Learn CoDEx-Large Dataset with NodePiece + RotatE as described by Galkin et al., 2022
+pipeline:
+  dataset: CoDExLarge
+  evaluator_kwargs:
+    filtered: true
+  loss: BCELoss
+  model: NodePiece
+  model_kwargs:
+    entity_initializer: xavier_uniform_
+    relation_initializer: init_phases
+    relation_constrainer: complex_normalize
+    embedding_dim: 200   # for both anchors and relations
+    interaction: RotatEInteraction
+    aggregation: mlp
+    tokenizers: ['AnchorTokenizer', 'RelationTokenizer']
```

Comment:
```suggestion
    tokenizers: ['anchor', 'relation']
```
You *could* drop the "Tokenizer" suffix

Reply:
It's supposed to be an 'official' config, let there be official names :)